### ISSUE TITLE
Augmentation while training

### ISSUE THREAD

**Issue by KatharinaSchmidt** [j96w/DenseFusion#172]
Hello, 

I want to ask if there is any augmentation (noise) to the rgb and depth-images while training? 
I just saw some noise added to target_t (position of object) and randomly change the brightness, contrast and saturation of rgb-image with ColorJitter. I want to add some simulated camernoise. 

Thanks for your answer

**Comment by Ixion46** [comment]
They mentioned these 3 Methods and I also did not found any more.

"To make the best use of the training set, several data augementation techniques are used in this code:

(1) A random noise is added to the brightness, contrast and saturation of the input RGB image with the torchvision.transforms.ColorJitter function, where we set the function as torchvision.transforms.ColorJitter(0.2, 0.2, 0.2, 0.05).

(2) A random pose translation noise is added to the training set of the pose estimator, where we set the range of the translation noise to 3cm for both datasets.

(3) For the YCB_Video dataset, since the synthetic data is not contain background. We randomly select the real training data as the background. In each frame, we also randomly select two instances segmentation clips from another synthetic training image to mask at the front of the input RGB-D image, so that more occlusion situations can be generated."
